cartographer_ros/launch/visualize_pbstream.launch

```diff
@@ -23,7 +23,7 @@
           -configuration_basenames visualize_pbstream.lua
           -keep_running=true
           -load_state_filename $(arg pbstream_filename)
-          -load_frozen_state=false"
+          -load_frozen_state=true"
```

Comment:
this seems to not load the constraints that you might want to visualize.

Reply:
ahh, I already wondered. The reason I set it to true is this revised check that would fail otherwise: https://github.com/magazino/cartographer_ros/blob/use-trajectory-states-interface/cartographer_ros/cartographer_ros/node.cc#L633

It used to use the `is_active_trajectory_` map before which I removed. Will see if we can find a workaround...